.github/push-to-main.yml

```
@@ -0,0 +1,44 @@
+name: push-to-main
+
+on:
+  push:
+    branches:
+      - master
```

I'll just rename the base branch to `main` now
```suggestion
      - main
```